### ISSUE TITLE
Workspace - System message for workspace description edit shows from "" to "x"

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#36595]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.42-1
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Navigate to staging.new.expensify.com
2. Create a new workspace.
3. Go to workspace profile.
4. Click Add description.
5. Add a description and save it.
6. Go to #admin room.

## Expected Result:
The system message shows <updated the description of this workspace to "m">.

## Actual Result:
The system message shows <updated the description of this workspace from "" to "x">.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/6b76c38c-801f-43a5-8779-d4ad34a9e7f3

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kbecciv** [comment]
We think that this bug might be related #wave5-free-submitters
CC @dylanexpensify

**Comment by Julesssss** [comment]
We should fix this, but I'm removing the blocker label as it shouldn't block the release 👍

**Comment by MitchExpensify** [comment]
Why would we expect this when the description was blank?:

> The system message shows <updated the description of this workspace to "m">.

**Comment by Julesssss** [comment]
> Why would we expect this when the description was blank?:

Oh right. Good point. @kbecciv any other issue here we're missing, or can we close?

**Comment by MitchExpensify** [comment]
> Oh right. Good point. @kbecciv any other issue here we're missing, or can we close?

Friendly bump @kbecciv

**Comment by kbecciv** [comment]
No other issues @MitchExpensify

**Comment by kbecciv** [comment]
@MitchExpensify Reopening the issue. 
Previously, we had an empty workspace description, but now there is a default workspace description (due to a new feature). However, the issue remains unresolved.
Could you please advise whether it's preferable to open a new issue or to keep this one open as it is?

https://github.com/Expensify/App/assets/93399543/b2831c59-489b-4fa3-94a5-7c1c91201241

**Comment by MitchExpensify** [comment]
Yeah lets solve that here, good catch. Can you please update the OP accordingly? Thanks

**Comment by abzokhattab** [comment]
Seems like it is a BE issue. the update description request returns the current format as a new report action in the response

**Comment by Julesssss** [comment]
@MitchExpensify this seems pretty low priority. Any preference for how we get this on the roadmap? I'll probably unassign once we've figured that out

**Comment by MitchExpensify** [comment]
I think its polish for Wave Collect

**Comment by situchan** [comment]
Unassigning as backend issue and no C+ required

**Comment by hungvu193** [comment]
unassigning myself as this no need C+

**Comment by MitchExpensify** [comment]
Will try track down an Internal volunteer as a next step

**Comment by MitchExpensify** [comment]
No bites yet

@trjExpensify do we ever apply hot picks label to polish?

**Comment by trjExpensify** [comment]
Yep, you can add the `Hot picks` label and move it into the `Hot picks` section of the project board so anyone browsing it can see it in that column. Once it's picked up, remove the label and move it back into `Polish`. 👍

**Comment by trjExpensify** [comment]
Why did you move this back, @MitchExpensify? @chiragsalian put up a PR here: https://github.com/Expensify/Web-Expensify/pull/41774

**Comment by chiragsalian** [comment]
Actually im just going to close out this issue since its done. Feel free to reopen if something is missing.
